README.md

```diff
@@ -190,7 +190,7 @@ be under the build directory you created.
 ```bash
 # Example on linux after running the build steps above. Assumes the
 # `benchmark` and `build` directories are under the current directory.
-$ g++ mybenchmark.cc -std=c++11 -isystem benchmark/include \
+$ g++ mybenchmark.cc -std=c++14 -isystem benchmark/include \
```

Comment:
this should be c++17 to be up to date with the library.

Reply:
Are there C++17 features in the header? If so, docs are outdated now too.
By the way, with cxx03 test gone, i guess we want to add a cxx14 test,
to ensure that the header can actually be used with the ${version} standard.

Reply:
Aha. Then it certainly might be great to "guard" against that with a test.